Source/Document Structure/SvgFragment.cs

```diff
@@ -171,9 +171,9 @@ protected override void Render(ISvgRenderer renderer)
                     var prevClip = renderer.GetClip();
                     try
                     {
-                        var size = this.Parent == null ? renderer.GetBoundable().Bounds.Size : GetDimensions();
-                        var clip = new RectangleF(this.X.ToDeviceValue(renderer, UnitRenderingType.Horizontal, this),
-                                                  this.Y.ToDeviceValue(renderer, UnitRenderingType.Vertical, this),
+                        var size = this is SvgDocument ? renderer.GetBoundable().Bounds.Size : GetDimensions(renderer);

```

Comment:
Is there a difference between `Parent == null` and `this is SvgDocument`?

Reply:
Outermost `<svg>` may also have `Parent`.(`Parent` is required to apply styles.)
So I changed condition.

Reply:
Ah, thanks - didn't see that!